docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/configs-syntax.md

```diff
@@ -1,17 +1,58 @@
-# Variables Syntax
+# CONFIG Syntax
 
 Nebula use `gflags` for run-time configurations.
 
-## Show variables
+The related three gflags parameters are: `rocksdb_db_options`,`rocksdb_column_family_options` and `rocksdb_block_based_table_options`.
```

Comment:
`, `

Reply:
I think the preview is good here.
![image](https://user-images.githubusercontent.com/42762957/67825396-29b9e380-fb04-11e9-834f-416ea2b716d3.png)